app/components/bali/progress/component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= tag.div(**options) do %>
+  <%= tag.progress(value: value, max: 100, style: @progress_bar_color) %>
```

Comment:
Im not sure, but I think `<progress>`  needs to have a closing tag. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Please make sure if `tag.progress` is closing automatically, also, the `percentage` can be nested on `<progress>`

Reply:
It does have a progress closing tag